README.md

```diff
@@ -43,7 +45,7 @@ cluster bootstrapping and initial payload deployment. In general, it is best
 to allow the OpenShift installer to manage its operation.
 
 To better understand how this operator is deployed, please see the
-[`manifest`](/manifest) directory. It contains a series of Kubernetes
+[`manifest`](/manifests) directory. It contains a series of Kubernetes
```

Comment:
since we are calling out the directory name explicitly, the link should have the same name
```suggestion
[`manifests`](/manifests) directory. It contains a series of Kubernetes
```

Reply:
Updated